src/Avalonia.Controls/Slider.cs

```diff
@@ -168,9 +169,11 @@ protected override void OnApplyTemplate(TemplateAppliedEventArgs e)
             _track = e.NameScope.Find<Track>("PART_Track");
             _increaseButton = e.NameScope.Find<Button>("PART_IncreaseButton");
 
+
             if (_track != null)
             {
                 _track.IsThumbDragHandled = true;
+                _track.KeyDown += TrackOnKeyDown;
```

Comment:
If the `Slider` template changes, this will leak subscriptions. We need to check for a non-null `track` and do a `_track.KeyDown -= TrackOnKeyDown` above (alongside where the other subscriptions are disposed) or use `AddDisposableHandler` to do the subscription.